CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- Fix incorrect game log attachment on Android ([#743](https://github.com/getsentry/sentry-unreal/pull/743))
```

Comment:
```suggestion
- On Android, the SDK no longer attaches the "wrong?" log file when capturing events ([#743](https://github.com/getsentry/sentry-unreal/pull/743))
```
The "wrong?" needs replacement or clarification. Or maybe not. Was the SDK just grabbing an older one?